liquidctl/driver/asus_gpu.py

```diff
@@ -28,8 +27,8 @@ class RogTuring(SmbusDriver):
     REG_BLUE = 0x05
     REG_GREEN = 0x06
     REG_MODE = 0x07
-    SYNC_REG = 0x0c     # unused
-    REG_APPLY = 0x0e
+    #SYNC_REG = 0x0c     # unused
+    #REG_APPLY = 0x0e    # unused
 
     _ASUS_GPU_MAGIC_VALUE = 0x1589
```

Comment:
Would this also need to be made public for testing?

Reply:
Not for now.

I'm using literals in the tests, they are:

- more easily matched against a SMBus capture
- able to get regressions like a misplaced edit of a register address (in the driver)

The idea is that to make the driver write wrongs things or the wrong places, at least two mistakes are necessary.

---

Arguably the same could be said about the vendor/device IDs, which are used in the probing logic and corresponding tests.

But I think this defense mechanism is less beneficial there: presumably the human would notice liquidctl no longer finding their device, and even if they didn't, for this mistake to affect others they would need a device with the wrong ID, and that device would also need to be substantially similar to the correct one (e.g. the wrong device must also use SMBus and in a way that doesn't break the driver in other places).

Also, the tests for probing logic are more readable with named constants.

Reply:
> able to get regressions like a misplaced edit of a register address (in the driver)

I had that issue already 😭 I swapped the green and blue registers addresses.

Reply:
Me too! : )